addon-test-support/focus.js

```diff
@@ -19,14 +19,23 @@ export function focus(selector) {
     let type = el.type;
     if (type !== 'checkbox' && type !== 'radio' && type !== 'hidden') {
```

Comment:
Based on that http://w3c.github.io/html/editing.html#the-tabindex-attribute it seems, it should be even more elements, that we actually can focus on, so the check, that includes only INPUT, contentEditable and A tag, is a bit too restricted.